DQM/L1TMonitor/src/L1TStage2CaloLayer1.cc

```diff
@@ -68,6 +68,17 @@ void L1TStage2CaloLayer1::dqmAnalyze(const edm::Event& event,
   bool caloLayer1OutOfRun{true};
   bool FATevent{false};
   bool additionalFB{false};
+  bool card7flag1354{false};
+  bool card7flag1356{false};
+  bool card7flag1358{false};
+  std::vector<uint32_t> FED1354_slot7_bits;
+  std::vector<uint32_t> FED1356_slot7_bits;
+  std::vector<uint32_t> FED1358_slot7_bits;
+  for (int i = 0; i < 6; i++) {
```

Comment:
I wonder if this recurring "6" could be more conveniently replaced by a `constexpr` parameter within the `dqmAnalyze` method

Reply:
we will consider that in the next round soon, thanks